packages/appium/docs/en/guides/migrating-1-to-2.md

```diff
@@ -184,231 +196,12 @@ TODO
-    "automationName": "XCUITest"
-  },
-  "$cloud:appiumOptions": {
-    "appiumVersion": "2.0.0",
-    "automationVersion": "3.52.0",
-    "plugins": ["images"]
-  }
-}
-```
-
-### Extension objects
-
-Some service providers may wish to dynamically allow access to all of the features of the Appium 2.0 CLI, including downloading arbitrary drivers and plugins. To represent these extensions, we can define special JSON "extension objects", with the following keys:
-
-- `name`: the name of the extension. This would be an NPM package name (if downloading from NPM), or a git or GitHub spec (if downloading from a git server or GitHub).
-- `version`: the version of the extension, e.g., the NPM package version or Git SHA.
-- (optional) `source`: a denotation of where the extension can be downloaded from. Recommended to support the following values: `appium`, `npm`, `git`, `github`. Here, `appium` means "Appium's own official list", and should be the default value if this key is not included.
-- (optional) `package`: when downloading extensions from git or github, the NPM package name of the extension must also be provided. This is optional for non-git sources.
-
-Since each session is handled by a single driver, the `$cloud:appiumOptions`/`$automation` capability could be used with an extension object value to denote this driver, for example:
-
-```json
-{
-    ...,
-    "$cloud:appiumOptions": {
-        ...,
-        "automation": {
-            "name": "git+https://some-git-host.com/custom-driver-project.git",
-            "version": "some-git-sha",
-            "source": "git",
-            "package": "driver-npm-package-name"
-        },
-        ...
-    },
-    ...
-}
-```
-
-And since sessions can handle multiple plugins, each value in the list of `$cloud:appiumPlugins` could also be an extension object rather than a string, so that specific versions could be requested:
-
-```json
-{
-    ...,
-    "$cloud:appiumOptions": {
-        ...,
-        "plugins": [{
-            "name": "images",
-            "version": "1.1.0"
-        }, {
-            "name": "my-github-org/my-custom-plugin",
-            "version": "a83f2e",
-            "source": "github",
-            "package": "custom-plugin"
-        }],
-        ...,
-    }
-    ...
-}
-```
-
-These serve as illustrative examples for the recommendations here. Of course it is up to the service providers to implement the handling of these capabilities at their front end / load balancer, to perform any error checking, or to actually run any of the `appium driver` or `appium plugin` CLI commands that support the end user's request. This section is merely a suggestion as to how service providers might design their user-facing capabilities API in a way which in principle supports all of the capabilities Appium itself would provide to the end user if they were running Appium on their own.
+We encourage cloud providers to thoroughly read and understand our [recommendation for cloud
```

Comment:
I don't think links can have linefeeds in them

Reply:
i think they can

Reply:
yep, i tested it in the docs site render and it works.